src/index.html

```diff
@@ -13,5 +13,151 @@
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+          method="post"
+          onsubmit="onSubmit()">
+      <div class="form-field" id="pers">
+
+        <fieldset class="form_regis">
+          <legend>Personal information
+          </legend>
+          <div class="form-field" id="registracia">
+      Username:
+            <input type="text"
+             name="username"
+             id="username">
+          </div>
+          <div class="form-field" id="name1">
+      Name:
+            <input type="text"
```

Comment:
```suggestion
          <div class="form-field" id="name1">
            Name:
            <input type="text"
```
Текст нужно располагать со сдвигом в 2 знака от тега